checkstyle/suppressions.xml

```diff
@@ -85,6 +85,9 @@
     <suppress checks="NPathComplexity"
               files="SslTransportLayer.java"/>
 
+    <suppress checks="ImportControl"
+              files="KafkaFuture.java|KafkaFutureTest.java"/>
```

Comment:
Note: I did add an entry to the `import-control.xml` for the `org.apache.kafka.clients.admin` namespace, which was previously missing.